topdown/parse_units.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2016 The OPA Authors.  All rights reserved.
```

Comment:
[nit] new file, let's make it 2022

Reply:
Resolved in style fixup commit from yesterday.